CHANGELOG.md

```diff
@@ -4,6 +4,33 @@ All notable changes to this project will be documented in this file.
 
 ## Unreleased
 
+## 0.7.0
+
+Released 2020-04-17
+
+#### :boom: Breaking Change
+* `opentelemetry-exporter-collector`
+  * [#901](https://github.com/open-telemetry/opentelemetry-js/pull/901) grpc for node and support for new proto format for node and browser ([@obecny](https://github.com/obecny))
+
+#### :bug: (Bug Fix)
+* `opentelemetry-plugin-http`
+  * [#960](https://github.com/open-telemetry/opentelemetry-js/pull/960) [http] fix: use url.URL ([@naseemkullah](https://github.com/naseemkullah))
+* `opentelemetry-plugin-express`
+  * [#910](https://github.com/open-telemetry/opentelemetry-js/pull/910) fix(plugin-express): fix double span end #908 ([@vmarchaud](https://github.com/vmarchaud))
```

Comment:
since the express moved i don't think we should add that commit, it will not be released without mirroring the release on the contrib

Reply:
Yes any plugin changes will now need to be in the contrib changelog

Reply:
See: https://github.com/open-telemetry/opentelemetry-js-contrib/pull/6

Reply:
I've approved the PR over the contrib repo. However i'm wondering if we need to keep version in sync between both, for example we don't have a major-level change on the contrib repo so the bump to 0.7 isn't exactly needed. Also what happens the other way around when there are no breaking changes on the main repo but one on the contrib ?

Reply:
I think for now we should release all packages as we know this works. We can open discussion about if versions should be kept sync or not.